pages/dev-mode.en.mdx

```diff
@@ -1,15 +1,30 @@
 # Dev mode 
 
-We have now added support to Figma's **Dev Mode** for the **Tokens Studio for Figma** plugin.
+We have    added support to Figma's **Dev Mode** for the **Tokens Studio for Figma** plugin.
```

Comment:
"Tokens Studio for Figma is integrated with Figma's Dev Mode"

-- Add some description here outlining the feature and what it does.